src/js/view/view.js

```diff
@@ -716,6 +724,17 @@ define([
                 default:
                     break;
             }
```

Comment:
Seems to me that we can merge the case statement above and the one below into one.